ocaml/parsing/parsetree.mli

```diff
@@ -338,6 +338,9 @@ and expression_desc =
              {{!expression_desc.Pexp_fun}[Pexp_fun]}.
            - [let f P = E] is represented using
              {{!expression_desc.Pexp_fun}[Pexp_fun]}.
+           - While Position arguments are parsed as
+             {{!Asttypes.arg_label.Labelled}[Labelled l]}, they are converted to
+             {{!Asttypes.arg_label.Position}[Position l]} arguments for type-checking.
```

Comment:
Good to mention `[%src_pos]` somewhere around here.

Reply:
Is my addition sufficient?